src/components/shared/form_fields/AmountField.vue

```diff
@@ -1,5 +1,8 @@
 <template>
 	<fieldset class="form-field form-field-amount" :class="`locale-${ $i18n.locale }`">
+		<div class="minimum-message">
```

Comment:
Because the `.minimum-message` div is going to be always ouput in this component now it'll add a gap even when there is no content because of the margin-bottom in the styles. In this case I would pass the message with a prop and use a v-if. You can see the extra gap here on the donation form:

<img width="444" alt="Screenshot 2023-10-24 at 08 35 49" src="https://github.com/wmde/fundraising-app-frontend/assets/663782/d5d20264-e52e-422d-94ca-03926839201d">